ts/components/services/ContactPreferencesToggles/index.tsx

```diff
@@ -55,7 +57,11 @@ const getChannelPreference = (
     pot.isSome(potServicePreference) &&
     isServicePreferenceResponseSuccess(potServicePreference.value)
   ) {
-    return potServicePreference.value.value[key];
+    if (key === "can_access_message_read_status") {
+      return potServicePreference.value.value[key] ?? true;
+    } else {
+      return potServicePreference.value.value[key];
+    }
```

Comment:
This condition looks a bit weird to me. Could we rewrite to make it clearer? Or at least leaving a comment to explain.

Reply:
You're right, this is actually useless since `can_access_message_read_status` it's not optional in that data structure, fixed in https://github.com/pagopa/io-app/pull/4011/commits/125d57a1c5c9e45f58758cde6137806e2a923872